Gemfile

```diff
@@ -11,20 +11,21 @@ end
 
 group :development do
   gem 'sqlite3'
+  gem 'web-console', '~> 2.0'
+  gem 'rails_db'
 end
 
 gem 'sass-rails', '~> 5.0'
 gem 'uglifier', '>= 1.3.0'
 gem 'coffee-rails', '~> 4.1.0'
 gem 'jquery-rails'
 gem 'turbolinks'
-gem 'bootstrap'
+gem 'bootstrap-sass'
 gem 'figaro'
 gem 'pry'
 
 group :development, :test do
   gem 'byebug'
-  gem 'web-console', '~> 2.0'
   gem 'spring'
   gem 'rspec-rails'
   gem 'shoulda'
```

Comment:
@ianroberts131 Further debugging-related gems to consider installing to help debugging efforts:
```
  # In development, test group:
  gem 'pry-rails'
  gem 'pry-inline' # automatically shows variable values inline in rails console
  gem 'rb-readline' # rb-readline makes pry history behave on all platforms (e.g. ensures ~/.pry_history auto loads)
```
```
  # In development group only
  gem 'better_errors' # Enhanced Rails error page with built-in rails console
  gem 'binding_of_caller' # Required by better_errors
  gem 'quiet_assets' # Less noisy but easier to debug server output
```